pkg/virt-operator/creation/rbac/operator.go

```diff
@@ -496,4 +497,4 @@ func GetKubevirtComponentsServiceAccounts(namespace string) []string {
 		fmt.Sprintf("%s:%s:%s", prefix, namespace, ApiServiceAccountName),
 		fmt.Sprintf("%s:%s:%s", prefix, namespace, ControllerServiceAccountName),
 	}
-}
\ No newline at end of file
+}
```

Comment:
looks like a redundant change... I think that the problem is that in your previous commit you changed this without a reason...